usaspending_api/etl/tests/integration/test_load_multiple_submissions.py

```diff
@@ -230,18 +230,18 @@ def setUp(self):
                     prior_year_adjustment
                 ) (values
                     (1, 1, 1, '1010', 'D', 1111, null, 'x'),
-                    (2, 1, 1, '1010', 'D', 2222, 'B', 'b'),
+                    (2, 1, 1, '1010', 'D', 2222, 'b', 'b'),
```

Comment:
The test data you have manipulated here is for loading File B. A good starting point for where you need to make a change is looking here: https://github.com/fedspendingtransparency/usaspending-api/blob/6f00957e15d24ec850d37438867437edfa3181a3/usaspending_api/etl/submission_loader_helpers/file_b.py#L51